gix-archive/Cargo.toml

```diff
@@ -33,7 +33,9 @@ gix-path = { version = "^0.10.14", path = "../gix-path", optional = true }
 gix-date = { version = "^0.9.3", path = "../gix-date" }
 
 flate2 = { version = "1.0.33", optional = true }
-zip = { version = "2.3.0", optional = true, default-features = false, features = ["deflate"] }
+zip = { version = "2.3.0", optional = true, default-features = false, features = [
```

Comment:
toml formatting from zed, I can remove it if you don't like it @Byron

Reply:
I like it and wish my toolchain would do that consistently. If `cargo fmt` would also deal with manifests I'd be very happy.